modules/rte_mis_mail/src/Form/SMSSettingsForm.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Drupal\rte_mis_mail\Form;
+
+use Doctrine\Common\Annotations\Annotation\Required;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\mobile_number\MobileNumberUtilInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a form to configure Mobile Number settings.
+ */
+class SMSSettingsForm extends ConfigFormBase {
+
+  /**
+   * The Mobile Number Utility service.
+   *
+   * @var \Drupal\mobile_number\MobileNumberUtilInterface
+   */
+  protected $mobileNumberUtil;
+
+  /**
+   * Constructs a MobileNumberSettingsForm object.
```

Comment:
It should be SmsSettingsForm

Reply:
Combined the forms and modified to email_sms_form.settings.